rocky/views/upload_csv.py

```diff
@@ -56,17 +59,24 @@ def get_context_data(self, **kwargs):
         context["criterias"] = CSV_CRITERIAS
         return context
 
-    def get_ooi_from_csv(self, ooi_type: str, value: str):
-        ooi = None
+    def get_network(self, network: str):
```

Comment:
This may be a bit of a violation since it's a getter method with side effects.

Perhaps it would be better to rename it to `get_or_create_network`:
```suggestion
    def get_or_create_network(self, network: str) -> Network:
```

Reply:
Agreed.